at_client/src/test/resources/org/atsign/virtualenv/docker-compose.yml

```diff
@@ -2,10 +2,11 @@ services:
   virtualenv:
     image: atsigncompany/virtualenv:dev_env
     ports:
-      - "127.0.0.1:6379:6379"
-      - "64:64"
-      - "127.0.0.1:9001:9001"
-      - "25000-25040:25000-25040"
+      - '127.0.0.1:6379:6379'
+      - '64:64'
+      - '443:443'
+      - '127.0.0.1:9001:9001'
+      - '25000-25999:25000-25999'
```

Comment:
no real reason to change but this is now in line with the docker-compose.yml in at_server/tools/virtualenv